sdk/core/core-autorest/api-extractor.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "types/latest/src/index.d.ts",
```

Comment:
Nit: Should this be:
```suggestion
  "mainEntryPointFilePath": "types/src/index.d.ts",
```
I don't think `latest` will exist until the types are rolled up.

Reply:
tsconfig puts them in latest directly: `"declarationDir": "./types/latest"`

Reply:
the downlevel compiler only creates the `types/3.1` folder, it doesn't touch `latest`

Reply:
Ah, I missed that. Nevermind then!